__tests__/ActionTrigger.test.ts

```diff
@@ -1,6 +1,4 @@
-import {App, Triggers, Args} from '../src/core/reactor';
-import {Origin, TimeValue} from '../src/core/time';
-import {Reactor, Timer, Action, Sched} from '../src/core/reactor';
+import {App, Triggers, Args, Origin, TimeValue, Reactor, Timer, Sched, Action} from '../src/core/internal';
```

Comment:
Tests should probably not use the internal exports but the external ones, which we yet have to set up.